src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2458,7 +2458,7 @@ mobile_apis::Result::eType ApplicationManagerImpl::CheckPolicyPermissions(
     const std::string& function_id,
     const RPCParams& rpc_params,
     CommandParametersPermissions* params_permissions) {
-  LOG4CXX_INFO(logger_, "CheckPolicyPermissions");
+  LOG4CXX_AUTO_TRACE(logger_);
   // TODO(AOleynik): Remove check of policy_enable, when this flag will be
   // unused in config file
```

Comment:
@BSolonenko Check if we still need this comment

Reply:
@AByzhynar  policy_enable used in 'smartDeviceLink.ini'